cmd/juju/status/output_tabular.go

```diff
@@ -177,6 +177,9 @@ func FormatTabular(writer io.Writer, forceColor bool, value interface{}) error {
 		if agentDoing != "" {
 			message = fmt.Sprintf("(%s) %s", agentDoing, message)
 		}
+		if u.Leader {
```

Comment:
how about we don't display this if there's only one unit? it's noisy/uninteresting in that case.